.github/workflows/linux.yml

```diff
@@ -0,0 +1,110 @@
+name: Linux
+
+on:
+  pull_request:
+  push:
+    branches:
+      - dev
+      - release
+
+jobs:
+  assert-test-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Install apt dependencies
+        run: |
+          sudo apt-get install --no-install-recommends \
```

Comment:
Is apt-get update no longer necessary here?

Reply:
Apparently not.
And if it is, we can always add it back.